google/cloud/spanner/internal/date.cc

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// To avoid secure warnings on Windows
```

Comment:
I think we can disable the warning for specific files in the CMake code instead:

https://github.com/googleapis/google-cloud-cpp/blob/84772e182e5a82c9f839a9073cef88b9962c48cf/google/cloud/storage/CMakeLists.txt#L243-L251

Reply:
Done